concepticondata/concepticon.tsv

```diff
@@ -2902,7 +2902,7 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 2905	KNIT	Basic actions and technology	Knitting is a method by which yarn is manipulated to create a textile or fabric for use in many types of garments.	Action/Process	
 2906	TRANSLATE	Basic actions and technology	To communicate of the meaning of a source-language text by means of an equivalent target-language text.	Action/Process	
 2907	ROT	Basic actions and technology	To be exposed to decomposition.	Action/Process	
-2908	SINK (DISAPPEAR IN WATER)	Basic actions and technology	To descend completely, leaving no trace in the surface.	Action/Process	
+2908	SINK (DISAPPEAR IN WATER)	Basic actions and technology	To go down in water, mud, quicksand etc.	Action/Process	
```

Comment:
So, the difference to the other SINK is that this one does not allow for agency?

Reply:
SINK (DESCEND) is the hyperonym. It is perhaps a misnomer as it includes both "cause something/someone else to sink and SINK (DISAPPEAR IN WATER)". We have list that clearly only refers to "cause to sink".